mim_ocr/graph/graph_utils.py

```diff
@@ -0,0 +1,55 @@
+from typing import Literal
+
+from igraph import Vertex
+
+from mim_ocr.data_model.box import Box
+from mim_ocr.graph.builders import EdgeBuilder
+
+Direction = Literal["left", "right", "up", "down"]
+
+
+def boxes_verdical_distance(b1: Box, b2: Box) -> float:
+    up = min(b1.top, b2.top)
+    down = max(b1.bottom, b2.bottom)
+
+    return down - up - b1.height() - b2.height()
+
+
+def boxes_horizontal_distance(b1: Box, b2: Box) -> float:
+    left = min(b1.left, b2.left)
+    right = max(b1.right, b2.right)
+
+    return right - left - b1.width() - b2.width()
+
+
+def get_neighbours(vertex: Vertex, direction: Direction, sorted: bool = True):
+    direction_function = None
+    match direction:
```

Comment:
flake8 protestuje przy tej składni, bo MIM_OCR wspiera także pythona 3.9, który jeszcze nie miał zdań match/case.

Reply:
Okej, to trochę zaskakujące w sumie. Zmieniłem na if casy.